frontend/lib/identification/activation_workflow/activation_code_scanner_page.dart

```diff
@@ -86,7 +88,7 @@ class ActivationCodeScannerPage extends StatelessWidget {
   ]) async {
     final client = GraphQLProvider.of(context).value;
     final projectId = Configuration.of(context).projectId;
-    final provider = Provider.of<UserCodeModel>(context, listen: false);
+    final userCodesModel = Provider.of<UserCodeModel>(context, listen: false);
```

Comment:
Shouldn't the class be called `UserCodesModel` (mind the new `s`).

Reply:
yes i thought about renaming everthing but the changes were hard to read when i change the name of the file

Reply:
Your choice, could also be done in a separate PR.